windows/RNGetRandomValues.vcxproj

```diff
@@ -0,0 +1,167 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build" ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(SolutionDir)packages\Microsoft.Windows.CppWinRT.2.0.190730.2\build\native\Microsoft.Windows.CppWinRT.props" Condition="Exists('$(SolutionDir)packages\Microsoft.Windows.CppWinRT.2.0.190730.2\build\native\Microsoft.Windows.CppWinRT.props')" />
```

Comment:
Having this point to the cppwinrt version in the SolutionDir might cause some issues, but there has been some active discussion on what we should be doing with these dependncies. @jonthysell might have a good idea here.

Reply:
This is correct. This makes sure you can both build the lib in local development, and when an app tries to consume it, the project will still build because the package will be restored relative to the app solution